src/proteus/utils/data.py

```diff
@@ -125,6 +125,21 @@ def download_stellar_spectra():
         log.info(f"Downloading stellar spectra to {data_dir}")
         download_folder(storage=storage, folders=[folder_name], data_dir=data_dir)
 
+def download_exoplanet_data():
+    """
+    Download exoplanet data
+    """
+    log.debug("Get exoplanet data?")
+
+    folder_name = 'Exoplanets'
+    storage = get_osf('fzwr4')
+
+    data_dir = GetFWLData() / "planet_reference"
+    data_dir.mkdir(parents=True, exist_ok=True)
+
+    if not (data_dir / folder_name).exists():
+        log.info(f"Downloading exoplanet population data to {data_dir}")
+        download_folder(storage=storage, folders=[folder_name], data_dir=data_dir)
```

Comment:
So this is the newest CSV file from DACE? Just to double check.

Reply:
Agree

Reply:
Maybe label it with the date you downloaded it (in case you haven't done this anyway), for future updates

Reply:
I put the date and other information inside the file